content/en/data_jobs/dbt.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Enable Data Jobs Monitoring for dbt
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+[Data Jobs Monitoring][1] gives visibility into the performance and reliability of dbt pipelines.
+
+## Requirements
+
+This guide is for [dbt-core][2] pipelines.
+
+1. [dbt v1.8][3] or later.
+2. [dbt-ol v1.26.0 ][4] or later.
+
+## Setup
+
+Follow these steps to enable dbt traces:
+
+1. Install `openlineage` provider by adding the following into your `requirements.txt`.
+    ```text
+    openlineage-dbt>=1.26.0
+    ```
+2. Configure `openlineage` provider. The simplest option is to set the following environment variables:
+    ```shell
+    export OPENLINEAGE_URL=<DD_DATA_OBSERVABILITY_INTAKE>
+    export OPENLINEAGE_API_KEY=<DD_API_KEY>
+    export OPENLINEAGE_NAMESPACE=<NAMESPACE>
+    ```
+
+  * Replace `DD_DATA_OBSERVABILITY_INTAKE` with `https://data-obs-intake.`{{< region-param key="dd_site" code="true" >}}.
+  * Replace `DD_API_KEY` with your valid [Datadog API key][5].
+  * Replace `NAMESPACE` if you want to use something other than the `default` namespace for job namespace.
+
+**Optional**
+
+1. Setup `OPENLINEAGE_DBT_LOGGING` environment variable, you can establish the logging level for the `openlineage.dbt` and its child modules.
```

Comment:
```suggestion
1. Set up the `OPENLINEAGE_DBT_LOGGING` environment variable. This will establish the logging level for the `openlineage.dbt` and its child modules.
```